controller/modules/gvpn/BaseTopologyManager.py

```diff
@@ -31,16 +32,15 @@ def __init__(self, CFxHandle, paramDict):
         #   self.links["on_demand"] = { uid: {"ttl": ttl, "rate": rate} }
         #   self.links["inbound"]   = { uid: None }
 
-        self.links = {}
-        self.links["successor"] = {}
-        self.links["chord"] = {}
-        self.links["on_demand"] = {}
-        self.links["inbound"] = {}
+        self.links = {
+            "successor": {}, "chord": {}, "on_demand": {}, "inbound": {}
+        }
 
         self.log_chords = []
 
-        # discovered nodes (via initial peer_state messages or advertisement)
+        # discovered nodes
         self.discovered_nodes = []
```

Comment:
it would be helpful to have a brief comment here that would make distinction between roles of two lists below.